projects/SelfTest/UsageTests/ToStringOptional.tests.cpp

```diff
@@ -0,0 +1,23 @@
+#define CATCH_CONFIG_ENABLE_OPTIONAL_STRINGMAKER
+#include "catch.hpp"
+
+#if defined(CATCH_CONFIG_CPP17_OPTIONAL)
+
+TEST_CASE( "std::optional<int> -> toString", "[toString][optional]" ) {
```

Comment:
The tests also need to be tagged with `[approvals]` tag, so that they are not checked by the approval tests.